src/Fhir.Anonymizer.FunctionalTests/CollectionResourceTests.cs

```diff
@@ -9,6 +11,7 @@ public class CollectionResourceTests
         private AnonymizerEngine engine;
         public CollectionResourceTests()
         {
+            FhirPathCompiler.DefaultSymbolTable.AddExtensionSymbols();
```

Comment:
Can we move this global initialization to a static constructor of Engine?

Reply:
Looks like we might not bind extension function to any class, and we can allow customer add some more.